class.jetpack-gutenberg.php

```diff
@@ -890,10 +890,10 @@ public static function validate_block_embed_url( $url, $allowed = array(), $is_r
 		// Normalize URL.
 		$url = sprintf(
 			'%s://%s%s%s',
-			$url_components['scheme'],
+			isset( $url_components['scheme'] ) ? $url_components['scheme'] : 'https',
```

Comment:
I think defaulting to https makes sense however is going to make that assumption possibly going to break any 3rd party embeds? I would think any external links would have the full scheme, but thinking this through further.

Reply:
I went with https because http would create insecure content warnings on http sites and everything we're working with would allow https.

If we're worried about this, we can default to whether the site itself is using https (`is_ssl`).

Reply:
I'd be okay with `https` as the default. I would assume that the blocks we're developing are all for services that are modern / popular enough to use HTTPS.